### PR TITLE
kyrgyz language added

### PR DESCRIPTION
**Pull request by yeraassyl**

### CODE REVIEW

File: src/main/java/pl/allegro/finance/tradukisto/ValueConverters.java [hunk 1]
```diff
@@ -3,20 +3,7 @@
...
-import static pl.allegro.finance.tradukisto.internal.Container.italianContainer;
-import static pl.allegro.finance.tradukisto.internal.Container.kazakhContainer;
-import static pl.allegro.finance.tradukisto.internal.Container.latvianContainer;
-import static pl.allegro.finance.tradukisto.internal.Container.polishContainer;
-import static pl.allegro.finance.tradukisto.internal.Container.russianContainer;
-import static pl.allegro.finance.tradukisto.internal.Container.serbianContainer;
-import static pl.allegro.finance.tradukisto.internal.Container.slovakContainer;
-import static pl.allegro.finance.tradukisto.internal.Container.turkishContainer;
-import static pl.allegro.finance.tradukisto.internal.Container.ukrainianContainer;
+import static pl.allegro.finance.tradukisto.internal.Container.*;
```
Comment (MaciejLipinski) [inline src/main/java/pl/allegro/finance/tradukisto/ValueConverters.java:6]: please revert to individual imports; we avoid star imports

File: src/test/groovy/pl/allegro/finance/tradukisto/ValueConvertersTest.groovy [hunk 2]
```diff
@@ -69,4 +69,9 @@ class ValueConvertersTest extends Specification {
         then:
         thrown(VerifyException)
     }
+
+    def "should convert number in Kyrgyz"() {
```
Comment (MaciejLipinski) [inline src/test/groovy/pl/allegro/finance/tradukisto/ValueConvertersTest.groovy:73]: `number` -> `numbers`
also please move this test above `should throw exception when null given`

File: src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/kyrgyz/KyrgyzValuesTest.groovy [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import spock.lang.Unroll
+
+import static pl.allegro.finance.tradukisto.internal.Container.kyrgyzContainer
+
+class KyrgyzValuesTest extends Specification {
+
+    static converter = kyrgyzContainer().getNumbersConverter()
+
+    @Unroll
+    def "should convert #value to '#words' in Kazakh"() {
```
Comment (MaciejLipinski) [inline src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/kyrgyz/KyrgyzValuesTest.groovy:13]: in Kyrgyz ;)

File: src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/kyrgyz/KyrgyzValuesTest.groovy [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        75_025        | "жетимиш беш миң жыйырма беш"
+        121_393       | "жүз жыйырма бир миң үч жүз токсон үч"
+        196_418       | "жүз токсон алты миң төрт жүз он сегиз"
+        317_811       | "үч жүз он жети миң сегиз жүз он бир"
+        514_229       | "беш жүз он төрт миң эки жүз жыйырма тогуз"
+        832_040       | "сегиз жүз отуз эки миң кырк"
+        1_346_269     | "бир миллион үч жүз кырк алты миң эки жүз алтымыш тогуз"
+        2_178_309     | "эки миллион жүз жетимиш сегиз миң үч жүз тогуз"
+
+        1_836_311_903 | "бир миллиард сегиз жүз отуз алты миллион үч жүз он бир миң тогуз жүз үч"
```
Comment (MaciejLipinski) [inline src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/kyrgyz/KyrgyzValuesTest.groovy:91]: I see you copied the test cases from `KazakhValuesTest`. I noticed it has values that differ from those present in other languages' tests. Please change these values to the ones that can be found eg. in `EnglishValuesTest`.
If you speak Kazakh, I would appreciate it very much if you changed the test cases in `KazakhValuesTest` as well :)

**Review by MaciejLipinski — CHANGES_REQUESTED** [review]
Thank you very much for your contribution! I merged the newest changes from master into your branch. There's a few things I'd like you to change before I can merge.
